pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/nullvalue/NullValueVectorCreator.java

```diff
@@ -42,7 +42,7 @@
  * null, which is true for all index creators types unless this one.
  */
 public class NullValueVectorCreator implements IndexCreator {
-  private final MutableRoaringBitmap _nullBitmap = new MutableRoaringBitmap();
+  private final RoaringBitmapWriter<RoaringBitmap> _bitmapWriter;
```

Comment:
Why the change to `RoaringBitmapWriter`?

Reply:
mainly to be consistent with inverted index creator, where such bitmap writer is used

Reply:
Understood, that makes a lot of sense.